cli/flox/src/commands/generations/rollback.rs

```diff
@@ -47,3 +50,45 @@ impl Rollback {
         Ok(())
     }
 }
+
+fn determine_previous_generation(
+    metadata: &AllGenerationsMetadata,
+) -> Option<(&GenerationId, &SingleGenerationMetadata)> {
+    metadata
+        .generations
+        .iter()
+        .sorted_by_key(|(_id, meta)| meta.last_active)
+        .rev()
+        .nth(1)
+}
+
+#[cfg(test)]
+mod tests {
+
+    use super::*;
+    use crate::commands::generations::test_helpers::mock_generations;
+
+    #[test]
+    fn rollback_uses_previously_active() {
+        // last created active
+        let metadata = mock_generations(3.into());
+        let Some((previous_generation, _metadata)) = determine_previous_generation(&metadata)
+        else {
+            panic!("expected to find previous generation")
+        };
+
+        assert_eq!(previous_generation, &2.into())
+    }
+
+    #[test]
+    fn rollback_rolls_back_to_newer_generation_if_previously_active() {
```

Comment:
Nit: the commit message says that this is about documenting the rollback sequence, but it's not actually documented in the code.